ui_framework/components/form/_color_picker.scss

```diff
@@ -0,0 +1,29 @@
+.kuiColorSwatch {
+  width: 19px;
+  height: 19px;
+  border-radius: 3px;
+  border: 1px solid black;
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.kuiColorPickerCover {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+
+.kuiColorPickerPopUp {
+  position: absolute;
+  z-index: 10;
+}
+
+.kuiColorLabel {
+  font-size: 16px;
```

Comment:
I think we should use $globalFontSize here instead, which will be more consistent with the font-size we're using elsewhere. This is reflected in the SCSS snippet in my comment.